build.sh

```diff
@@ -16,6 +16,7 @@ export MINGW_BRANCH=${MINGW_BRANCH:-woarm64}
 
 export RUN_BOOTSTRAP=${RUN_BOOTSTRAP:-1}
 export UPDATE_SOURCES=${UPDATE_SOURCES:-1}
+export RESET_SOURCES=${RESET_SOURCES:-1}
```

Comment:
I suggest to remove RESET_SOURCES from here. It will be 0 be default.
It will prevent overwriting changes locally incidentally.
if it is needed for a workflow, it can be defined there.

Reply:
It's needed for this script that is used locally, not for the workflow.

Reply:
Ah, now I see it, that's just copy&paste mistake...